scripts/update-bootstrap.ts

```diff
@@ -0,0 +1,50 @@
+import assert from 'assert';
+
+import cliPkg from '@crackle/cli/package.json';
+import bootstrapPkg from '@crackle-private/bootstrap/package.json';
+
+import { run, done, fromRoot } from './utils';
+
+const distTags = JSON.parse(
+  `${await run(`npm info @crackle/cli dist-tags --json`, {
+    encoding: 'utf-8',
+    stdio: 'pipe',
+  })}`,
+) as Record<string, string | undefined>;
+
+let version = distTags.bootstrap;
+
+if (!version) {
+  console.log('No bootstrap tag found. Getting latest version...');
+
+  const latest = `${await run(`npm info @crackle/cli@latest version`, {
+    encoding: 'utf-8',
+    stdio: 'pipe',
+  })}`.trim();
+
+  version = latest;
+}
+
+assert(
+  version !== cliPkg.version,
+  'Attempting to update bootstrap with workspace version',
+);
+
+if (version === bootstrapPkg.dependencies['@crackle/cli']) {
+  console.warn(`@crackle/cli@${version} already installed`);
+
+  // eslint-disable-next-line no-process-exit
+  process.exit(0);
+}
```

Comment:
Will this always be false since [the registry is in the package version](https://github.com/seek-oss/crackle/blob/3c07ab9b45f04db62b68cc2aac553c021f70a5d8/bootstrap/package.json#L8)?

Reply:
The registry will be gone once this PR goes through. Using `npm:` is too finicky and won't be needed anyway.